internal/push/push.go

```diff
@@ -199,10 +200,7 @@ func (s *subscription) listen(ctx context.Context) {
 
 	for {
 
-		if err = s.connect(ctx, !isReconnection); err != nil {
-			s.publishError(err)
```

Comment:
hum you removed the publishError completely here

Reply:
`err` was always set to nil, so this was always an unreachable code path.